common/src/main/java/com/wynntils/features/ui/WynncraftButtonFeature.java

```diff
@@ -66,6 +71,22 @@ private void addWynncraftButton(TitleScreen titleScreen) {
         titleScreen.addRenderableWidget(wynncraftButton);
     }
 
+    /* connects to the server on "startup" - off by default */
+    private void autoConnect() {
+        if (!initialBoot && autoConnect.get()) {
+            ServerData wynncraftServer =
+                    new ServerData("Wynncraft", connectToLobby.get() ? LOBBY_SERVER : GAME_SERVER, false);
+            wynncraftServer.setResourcePackStatus(ServerData.ServerPackStatus.ENABLED);
```

Comment:
Has this been checked in both cases where the mod loads the resource pack, and where the mod does not?